audit/projects/k8s-conform/service-accounts/service-cri-o@k8s-conform.iam.gserviceaccount.com/description.json

```diff
@@ -1,5 +1,5 @@
 {
-  "displayName": "service-cri-o",
+  "displayName": "Grants write access to gs://k8s-conform-cri-o",
```

Comment:
Expected, k8s-conform service-account description changes are due to https://github.com/kubernetes/k8s.io/pull/1890#issuecomment-816326889